src/libs/antares/CMakeLists.txt

```diff
@@ -518,6 +517,8 @@ target_link_libraries(libantares-core
 		correlation
 		object
         benchmarking
+	PRIVATE
+		Antares::date

```

Comment:
I don't think it should be private:
some date classes are used in Study class, so any user of the lib will also need the date headers.